build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        springBootVersion = '2.3.4.RELEASE'
+        springBootVersion = '2.4.4'
```

Comment:
⚠️ Updating to Spring Boot 2.4 will require further changes to move to the reading of secrets (same changes as per Slack discussion).
https://spring.io/blog/2020/08/14/config-file-processing-in-spring-boot-2-4